src/web/mjs/connectors/ReaperScansID.mjs

```diff
@@ -0,0 +1,14 @@
+import ReaperScans from './ReaperScans.mjs';
+
```

Comment:
It wont work. ReaperScansID is using madara, while ReaperScans is totally different code.

Reply:
ReaperscansFR work use reaperscans

Reply:
Wrong. Check this pending PR.

https://github.com/manga-download/hakuneko/pull/5309/files#diff-25126efd42997146768e026d5a02d7f10ba9adf642c8ed2d20d97f4696e10403

FR uses Madara. Main website dont. It was the purpose of the fix

Reply:
![image](https://user-images.githubusercontent.com/10671518/210272770-72741896-477a-4ee6-834c-770c422ed628.png)

Not getting chapters. Just follow my advices :D

>Just extend Madara and change queryChapters and queryChapterTitleBloat, it works

Reply:
Ok